salt/modules/keyboard.py

```diff
@@ -77,6 +78,11 @@ def set_sys(layout):
         __salt__["file.sed"](
             "/etc/conf.d/keymaps", "^keymap=.*", "keymap={}".format(layout)
         )
+
+
+    set_sys_warning_message = "The set_sys function will return True instead of layout in the Argon release"
```

Comment:
Can we mark these as comments. Warning here would be far too noisy. Marking these as comments will allow our team to know this code needs to be changed for the Argon release. And the notice for users will be the changelog.

Reply:
Done! :)